db/schema.rb

```diff
@@ -25,6 +25,16 @@
     t.index ["user_id"], name: "index_comments_on_user_id"
   end
 
+  create_table "friendships", force: :cascade do |t|
+    t.bigint "requester_user_id", null: false
+    t.bigint "reciever_user_id", null: false
```

Comment:
**Optional** : If you keep one of the columns as user_id and the other as `friend_id` the associations are easier to set and read:
`belongs_to :user`
`belongs_to :friend, class_name: "User"`